config.js

```diff
@@ -14,6 +14,16 @@ function getConfig() {
     const releaseNotesPrefix = core.getInput('release-notes-prefix');
     const releaseNotesSuffix = core.getInput('release-notes-suffix');
 
+    let releaseNotesPrefixPattern;
+    if (releaseNotesPrefix !== undefined && releaseNotesPrefix !== "") {
```

Comment:
nit: use single quotes for everything except for when you have a specific need to escape special chars or perform substitution.

Reply:
Fixed.